Adyen.Test/Adyen.Test.csproj

```diff
@@ -5,11 +5,11 @@
 
     <IsPackable>false</IsPackable>
 
-    <AssemblyVersion>4.0.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0</AssemblyVersion>
```

Comment:
breaking change because of multibanco. follow here other upgrades for example php https://github.com/Adyen/adyen-java-api-library/releases/tag/3.0.0

Reply:
Do you mean the action property ? It is already there
https://github.com/Adyen/adyen-dotnet-api-library/blob/0366db84d7111281b3572f5980a45e2f70768515/Adyen.Test/CheckoutTest.cs#L514

Reply:
Can you check again ? @Aleffio  @rikterbeek